test/Feature/QuerySpec.hs

```diff
@@ -395,6 +396,76 @@ spec actualPgVersion = do
+
+          get "/reference_from_partitioned?select=id,partitioned_a(id,name)&order=id.asc" `shouldRespondWith`
+            [json|
+              [{"id":1,"partitioned_a":[{"id":1,"name":"first"}]},
+               {"id":2,"partitioned_a":[]}] |]
+            { matchHeaders = [matchContentTypeJson] }
+
+        when (actualPgVersion >= pgVersion121) $ do
+          it "can request parents and children between partitioned tables" $ do
+            get "/partitioned_a?id=in.(1,2,4)&select=id,name,partitioned_b(id,name)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":1,"name":"first","partitioned_b":[]},
+                 {"id":2,"name":"first","partitioned_b":[{"id":2,"name":"first_b"}]},
+                 {"id":4,"name":"second","partitioned_b":[{"id":4,"name":"second_b"}]}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+            get "/partitioned_b?id=in.(2,4)&select=id,name,partitioned_a(id,name)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":2,"name":"first_b","partitioned_a":{"id":2,"name":"first"}},
+                 {"id":4,"name":"second_b","partitioned_a":{"id":4,"name":"second"}}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+          it "can request parents and children between a partitioned table and a partition" $ do
+            get "/partitioned_a?id=in.(1,2,4)&select=id,name,first_partition_b(id)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":1,"name":"first","first_partition_b":[]},
+                 {"id":2,"name":"first","first_partition_b":[{"id":2}]},
+                 {"id":4,"name":"second","first_partition_b":[]}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+            get "/first_partition_b?select=id,name,partitioned_a(id,name)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":1,"name":"first_b","partitioned_a":null},
+                 {"id":2,"name":"first_b","partitioned_a":{"id":2,"name":"first"}}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+            get "/partitioned_b?id=in.(1,3,4)&select=id,name,second_partition_a(id,name)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":1,"name":"first_b","second_partition_a":null},
+                 {"id":3,"name":"second_b","second_partition_a":null},
+                 {"id":4,"name":"second_b","second_partition_a":{"id":4,"name":"second"}}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+            get "/second_partition_a?select=id,name,partitioned_b(id,name)&order=id.asc" `shouldRespondWith`
+              [json|
+                [{"id":3,"name":"second","partitioned_b":[]},
+                 {"id":4,"name":"second","partitioned_b":[{"id":4,"name":"second_b"}]}] |]
+              { matchHeaders = [matchContentTypeJson] }
+
+          it "can request parents and children between a table and a partitioned table" $ do
```

Comment:
From a glance, this looks the same as the above test that says:

```haskell
it "can request parents and children between a partitioned table and a table"
```

But the difference is in the target of the foreign key right? Perhaps you can clarify that on the specs.